iulianIachimov/src/main/java/dataCollectionTask/Models/AbstractOrder.java

```diff
@@ -0,0 +1,100 @@
+package dataCollectionTask.Models;
+
+import dataCollectionTask.interfaces.Order;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public abstract class AbstractOrder implements Order {
+  private final String id;
+  private final LocalDateTime date;
+  private List<Item> items;
+
+  public AbstractOrder(List<Item> items) {
+    this.id = id();
```

Comment:
I would like to recomment you the following: this.id = UUID.randomUUID();
and remove the method id();

Reply:
Thank you for recommendation, it was implemented.